configure.ac

```diff
@@ -1394,11 +1394,7 @@ AS_IF([test `uname -s` = "Linux" -o `uname -s` = "FreeBSD" -o `uname -s` = "Darw
       # optional. If node and npm are not available, cool will have to be built on a Linux box.
       [AC_PATH_PROG(NODE, node, no)
        if test "$NODE" = "no"; then
-           if test `uname -s` = "Darwin"; then
-               AC_MSG_WARN([The cool bits will have to be built on a Linux machine and copied over])
-           else
-               AC_MSG_ERROR([node required to build cool, but not installed])
-           fi
+            AC_MSG_ERROR([node required to build cool, but not installed, once installed replace 'no' with actual node path in Makefile])
```

Comment:
I don't think Makefile with `no` would be generated. I think configure script will stop at error. Hence the `once installed replace 'no' with actual node path in Makefile` part is not necessary.

Reply:
I got below error even after installing the node
`Checking for CSS errors...
/bin/sh: no: command not found
make[1]: *** [Makefile:1100: dist/bundle.css] Error 127
make[1]: Leaving directory '/Users/karthik/Desktop/online/browser'
make: *** [Makefile:709: all] Error 2`

And error got resolved after replacing `no` with actual path of the node in make file. Hence made these changes.

Reply:
Right, you need to `make distclean` after installing node, configure assumes the system doesn't change after running once. In any case, could you please amend your commit message to include the signed-off-by line as requested above? We can take care of the rest. Thanks.

Reply:
Done